acronyms.csv

```diff
@@ -3284,6 +3284,7 @@ OIS,Office of Information Security,,
 OI&T,Office of Information & Technology,VA dept,
 OITC,Office of Information Technology and Communications,,
 OITSC,OI Tactical Space Committee,,
+OKR,Objectives Keys & Results,,
```

Comment:
Looks like it is [`Objectives and Key Results](https://felipecastro.com/en/okr/what-is-okr/) after all.

Reply:
```suggestion
OKR,Objectives and Key Results,,
```